pkg/api/types.go

```diff
@@ -3109,6 +3109,8 @@ const (
 	NodeMemoryPressure NodeConditionType = "MemoryPressure"
 	// NodeDiskPressure means the kubelet is under pressure due to insufficient available disk.
 	NodeDiskPressure NodeConditionType = "DiskPressure"
+	// NodeCPUPressure means the kubelet is under pressure due to insufficient available cpu.
```

Comment:
This was necessary because we decided to drive reservations solely by system- and kube-reserved. We considered reserving threads for the shared pool, but it has unfortunate implications for CPU accounting. The current strategy favors pods with a CPU request. A low-water-mark (either by config or unconditional) would favor pods without a CPU request since G/exclusive pods may have to be bounced after scheduling.